airflow/ui/src/pages/TaskInstance/Header.tsx

```diff
@@ -46,9 +46,13 @@ export const Header = ({
     { label: "Start", value: <Time datetime={taskInstance.start_date} /> },
     { label: "End", value: <Time datetime={taskInstance.end_date} /> },
     { label: "Duration", value: `${getDuration(taskInstance.start_date, taskInstance.end_date)}s` },
-    ...(taskInstance.dag_version?.version_number === undefined
-      ? []
-      : [{ label: "Dag Version", value: `v${taskInstance.dag_version.version_number}` }]),
+    {
+      label: "DAG Version at time of run",
```

Comment:
```suggestion
      label: "DAG Version",
```

Reply:
Same that was on the figma, but glad to update.

Reply:
Done